server/api/src/rsDataObjPut.cpp

```diff
@@ -669,6 +669,8 @@ int preProcParaPut(
     return status;
 } // preProcParaPut
 
+int crash(int* t) { return 5 + *t; }
```

Comment:
why 5?  give it a name?  or comment.

Reply:
This code will be removed. It's there to trigger a stacktrace and see if things work.

Reply:
ohright.  i read that.

Reply:
lol

Reply:
will test what the stacktrace output looks like using this.